lib/stack_master/parameter_resolver.rb

```diff
@@ -42,7 +42,7 @@ def load_parameter_resolver(class_name)
     end
 
     def resolve_parameter_value(parameter_value)
-      return parameter_value if String === parameter_value || parameter_value.nil?
+      return parameter_value if Numeric === parameter_value || String === parameter_value || parameter_value.nil?
```

Comment:
Could we move this to a separate line with something like this? `return parameter_value.to_s if Numeric === parameter_value`